frontend/src/modules/entitieslist/components/Entity.js

```diff
@@ -20,6 +20,8 @@ type Props = {
     selected: boolean,
     selectEntity: Function,
     getSiblingEntities: Function,
+    areFiltersApplied: Function,
```

Comment:
You can remove this line. `areFiltersApplied` is not passed as a prop from another component. It's a function **within** the component.

Reply:
I see that what was causing the error for Flow `Cannot create Entity element because property areFiltersApplied is missing in props [1] but exists in Props [2].`